src/reducers/eventTypeReducer.js

```diff
@@ -14,6 +14,8 @@ export const records = (state = defaultEventTypeCollection, action) => {
     switch(action.type){
         case eventTypeActions.GET_EVENT_TYPE_SUCCESS:
             return mergeToStateById(state, action.eventType)
+        case eventTypeActions.GET_EVENT_TYPES_SUCCESS:
```

Comment:
Use the multiple case for a single op syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#Multi-case_-_single_operation

Reply:
These do different things.    One works with a single eventType and the other works with multiple eventTypes.